src/main/java/city/parking/services/ParkingCostCalculationUtil.java

```diff
@@ -0,0 +1,56 @@
+package city.parking.services;
+
+import city.parking.entities.Money;
+import city.parking.entities.ParkingProcess;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Currency;
+
+public class ParkingCostCalculationUtil {
```

Comment:
To brzmi jak nie Util a pełnoprawna klasa biznesowa - dobry kandydat na jakiś serwis.